### PR TITLE
Fix #22

### LINKED ISSUE #22: Update username based commands (!addaction, !display for now) to allow usernames with spaces
currently, if adding action to user 'cool-ass username', Steven will assume that the username is just 'cool-ass'

### PR DESCRIPTION
**Pull request by Jerrgree**
Fixes #22 by setting the last argument as a dynamic list and joining it to construct user information.

### CODE REVIEW

**Comment by brooksbecton** [comment]
`oof`

File: lib/Steven/admin.rb [hunk 1]
```diff
@@ -2,18 +2,19 @@ module Steven
   # Commands available to the user set as `owner_id` in config.yml
   module Admin
     extend Discordrb::Commands::CommandContainer
-    command :addaction do |event, action, user_info|
+    command :addaction do |event, action, *user_info|
+      @user = user_info.join(" ")
```
Comment (laurenball) [inline lib/Steven/admin.rb]: I'm not crazy about making a user variable that doesn't point to a user object

**Comment by laurenball** [comment]
> `oof`

`oof`

File: lib/Steven/admin.rb (hunk 1 above)
Comment (Jerrgree) [reply lib/Steven/admin.rb]: Updated to use reuse variable

**Comment by laurenball** [comment]
thanks!

**Pull request merged**